metrics/web/test/project_groups/presentation/widgets/project_group_card_test.dart

```diff
@@ -83,13 +122,105 @@ void main() {
     );
 
     testWidgets(
-      "hides the edit button if the widget is not hovered",
+      "applies the border color from the metrics theme to the border side of the padded card",
       (WidgetTester tester) async {
         await tester.pumpWidget(const _ProjectGroupCardTestbed(
           theme: testTheme,
           projectGroupCardViewModel: projectGroupCardViewModel,
         ));
 
+        final paddedCard = tester.widget<PaddedCard>(find.byType(PaddedCard));
+        final borderShape = paddedCard.shape as RoundedRectangleBorder;
+
+        expect(borderShape.side.color, equals(testBorderColor));
+      },
+    );
+
+    testWidgets(
+      "applies the title style from the metrics theme to the project group card view model name text",
```

Comment:
```suggestion
      "applies the title style from the metrics theme to the name text",
```
what if we shorten it like this? (here and for others) - will it improve readability?